pkg/dns/ibm/dns.go

```diff
@@ -139,9 +143,18 @@ func (p *Provider) createOrUpdateDNSRecord(record *iov1.DNSRecord, zone configv1
 	if !ok {
 		return fmt.Errorf("createOrUpdateDNSRecord: unknown zone: %v", zone.ID)
 	}
+
+	// TTL must be between 120 and 2,147,483,647 seconds, or 1 for Automatic.
+	if (record.Spec.RecordTTL > 1) && (record.Spec.RecordTTL < 120) {
```

Comment:
Is 0 a valid value for RecordTTL?

Reply:
If RecordTTL is 0 it doesn't cause an issue. CIS will convert it to 1 (Automatic). Tested it by setting RecordTTL to 0:
```
2021-09-08T12:30:45.657Z	INFO	operator.dns	ibm/dns.go:93	updated DNS record	{"record": {"dnsName":"*.apps.ipi-ingress.szucsati-ipi.com.","targets":["1e1f0db9-us-east.lb.appdomain.cloud"],"recordType":"CNAME","recordTTL":0}, "zone": {"id":"826e86e6067eef5f80330f6692a1dcea"}, "target": "1e1f0db9-us-east.lb.appdomain.cloud"}
```
output from CIS:
```
 $ ibmcloud cis dns-records 826e86e6067eef5f80330f6692a1dcea
Getting DNS Records in domain '826e86e6067eef5f80330f6692a1dcea' for service instance 'ipi-cis' ...
OK
ID                                 Name                                        Type    Content                               Proxied   TTL
1eb68061c11ece6d028f8e572dc996b8   *.apps.ipi-ingress.szucsati-ipi.com         CNAME   1e1f0db9-us-east.lb.appdomain.cloud   N/A       1
```